core/register_core_types.cpp

```diff
@@ -145,10 +146,12 @@ void register_core_types() {
 	GDREGISTER_CLASS(Resource);
 	GDREGISTER_CLASS(Image);
 
+	GDREGISTER_CLASS(Shortcut);
 	GDREGISTER_VIRTUAL_CLASS(InputEvent);
 	GDREGISTER_VIRTUAL_CLASS(InputEventWithModifiers);
 	GDREGISTER_VIRTUAL_CLASS(InputEventFromWindow);
 	GDREGISTER_CLASS(InputEventKey);
+	GDREGISTER_CLASS(InputEventShortcut);
```

Comment:
Exposing InputEventShortcut is a bit useless without exposing the `shortcut` property.

Reply:
done